google-cloud-clients/google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java

```diff
@@ -629,7 +632,7 @@ public void onFailure(Throwable t) {
   static ListLogEntriesRequest listLogEntriesRequest(
       String projectId, Map<Option.OptionType, ?> options) {
     ListLogEntriesRequest.Builder builder = ListLogEntriesRequest.newBuilder();
-    builder.addProjectIds(projectId);
+    builder.addResourceNames(projectId);
```

Comment:
I just had a look at this Builder's [Javadoc](https://github.com/googleapis/google-cloud-java/blob/master/google-api-grpc/proto-google-cloud-logging-v2/src/main/java/com/google/logging/v2/ListLogEntriesRequest.java#L1002-L1006): 

> **these project identifiers are converted to resource name format** 

This line sounds like we have to convert the `projectId` before adding it in `resourceName`. Please confirm with someone who works/worked for this client.

cc:@ajaaym

Reply:
ok i will conform and than update